lib/rubocop/cop/mixin/space_after_punctuation.rb

```diff
@@ -19,9 +19,14 @@ def investigate(processed_source)
         end
       end
 
+      def space_style_before_rcurly?
+        throw RuntimeError.new(
```

Comment:
It should be `raise`, not `throw` by the way, but like I said, it should be removed IMO.